src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -435,18 +442,48 @@ public void Clear()
             }
         }
 
-        // Having lots of extra slime jets and mucocysts really doesn't help you that much
+        // Having lots of extra pili, slime jets and mucocysts doesn't really help much
+        pilusScore *= MathF.Sqrt(pilusCount);
         slimeJetScore *= MathF.Sqrt(slimeJetsCount);
-        slimeJetScore *= slimeJetsMultiplier;
-
         mucocystsScore *= MathF.Sqrt(mucocystsCount);
 
+        slimeJetScore *= slimeJetsMultiplier;
+
         var predationToolsRawScores = (pilusScore, oxytoxyScore, slimeJetScore, mucocystsScore);
 
         cachedPredationToolsRawScores.Add(microbeSpecies, predationToolsRawScores);
         return predationToolsRawScores;
     }
 
+    public (float EnzymesScore, bool IsMembranedigestable) GetEnzymesScores(MicrobeSpecies predator,
```

Comment:
Is there a reason why this doesn't just return 0 when indigestible?

Reply:
The engulfment score is muliplied by enzymes score so its default value is 1, if that's what you mean

Reply:
No, I meant that as there's only use of the score inside a check with the bool, you might as well just return 0 if the bool would be false. In fact I've done that change and I'll push what I have in a minute or so for you to look at.

Reply:
but maybe you're right and I should return only 0 if IsMembranedigestable is false , otherwise return the enzyme score